index.html

```diff
@@ -1,10 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
+<h1>Моя первая страница</h1>
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>Title</title>
-</head>
-<body>
+  </head>
```

Comment:
Open and close tags should have the same padding.

Reply:
Hereinafter use prettier before commit.

Reply:
Ok